core/lib/data/utils/compress_file_utils.dart

```diff
@@ -0,0 +1,40 @@
+
+import 'dart:typed_data';
+
+import 'package:core/utils/app_logger.dart';
+import 'package:core/utils/build_utils.dart';
+import 'package:flutter_image_compress/flutter_image_compress.dart';
+
+class CompressFileUtils {
+
+  /// Maximum image size displayed (Unit: KB)
+  static const int MAXIMUM_IMAGE_SIZE = 300;
+  /// Quality image 0 - 100
+  static const int QUALITY_DEFAULT = 50;
+
+  bool _exceedMaximumImageSize(Uint8List bytesData) {
+    final maximumSizeBytes = MAXIMUM_IMAGE_SIZE * 1024;
+    final sizeBytesData = bytesData.length;
+    log('DownloadClient::exceedMaximumImageSize(): maximumSizeBytes: $maximumSizeBytes');
+    log('DownloadClient::exceedMaximumImageSize(): sizeBytesData: $sizeBytesData');
+    return sizeBytesData > maximumSizeBytes;
+  }
+
+  Future<Uint8List> compressBytesDataImage(Uint8List bytesData, {int? maxWidth}) async {
+    if (_exceedMaximumImageSize(bytesData)) {
+      log('DownloadClient::_compressImageData(): BEFORE_COMPRESS: bytesData: ${bytesData.lengthInBytes}');
+      if (BuildUtils.isWeb) {
+        return bytesData;
```

Comment:
because we compressed when picked image, so no need to compress at this point? Am I right?

Reply:
if yes, how the picker compress image for web? please address it in `ADR` also.

Reply:
As your previous comment. I canceled all web operations. Web does not compress anything

Reply:
I agreed with the compress in picking step. dont' remove it

Reply:
ok remove compress for web